crossdock/thrift/gen-go/echo/echo.go

```diff
@@ -0,0 +1,418 @@
+// Autogenerated by Thrift Compiler (0.9.3)
+// DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
```

Comment:
For all code in the `gen-go` directory, that's presumably generated by running thrift-gen? Can you add a go:generate entry to `crossdock/thrift/gen.go` so that we can just do `go generate` in that directory to generate both, the thriftrw and the Apache Thrift code?